nixos/tests/seatd.nix

```diff
@@ -0,0 +1,58 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+let
+  seatd-test = pkgs.writeTextFile {
```

Comment:
This looks like what `writeShellScriptBin` is doing already.
https://nixos.org/manual/nixpkgs/unstable/#trivial-builder-writeText

Reply:
I think [`writeShellApplication`](https://ryantm.github.io/nixpkgs/builders/trivial-builders/#trivial-builder-writeShellApplication) would be even better. Adding a shellcheck to the script makes it safer to hack on later.

Reply:
I think [`writeShellApplication`](https://ryantm.github.io/nixpkgs/builders/trivial-builders/#trivial-builder-writeShellApplication) would be even better. Adding a shellcheck to the script makes it safer to hack on later.